tools/issue_transaction.cpp

```diff
@@ -0,0 +1,162 @@
+#include "helper/issue_transaction_update.hpp"
+#include "helper/issue_transaction_remove.hpp"
+#include "helper/issue_transaction_transfer.hpp"
+
+
+namespace tools {
+    namespace issue_transaction {
+        const std::string kADD = "add";
+        const std::string kTRANSFER = "transfer";
+        const std::string kUPDATE = "update";
+        const std::string kREMOVE = "remove";
+        const std::string kCONTRACT = "contract";
+
+        const std::string kASSET = "asset";
+        const std::string kDOMAIN = "domain";
+        const std::string kACCOUNT = "account";
+        const std::string kPEER = "peer";
+
+        std::string tolower(std::string s) {
+            for (char &c : s)
+                c = std::tolower(c);
+            return s;
+        }
+
+        void invalid_error() {
+            logger::error("issue_transaction") << ("Invalid argument");
+            logger::error("issue_transaction") << ("./issue_transaction [COMMAND] [DATAMODEL] [ARGS...] ");
+            exit(1);
+        }
+
+        void undefined_data_error( std::string& data ) {
+            logger::error("issue_transaction") << data << " is undefined data";
+            exit(1);
+        }
+
+
+
+        void Add( std::string& data, std::vector<std::string>& argv ) {
+            if( data == kASSET ) {
+                add::asset::issue_transaction( argv );
+            } else if( data == kDOMAIN ) {
+                add::domain::issue_transaction( argv );
+            } else if( data == kACCOUNT ) {
+                add::account::issue_transaction( argv );
+            } else if( data == kPEER ) {
+                add::peer::issue_transaction( argv );
+            } else {
+                undefined_data_error( data );
+            }
+        }
+        void Transfer( std::string& data, std::vector<std::string>& argv ) {
+            if( data == kASSET ) {
+                add::asset::issue_transaction( argv );
+            } else if( data == kDOMAIN ) {
+                add::domain::issue_transaction( argv );
+            } else if( data == kACCOUNT ) {
+                add::account::issue_transaction( argv );
+            } else if( data == kPEER ) {
+                add::peer::issue_transaction( argv );
+            } else {
+                undefined_data_error( data );
+            }
+        }
+        void Update( std::string& data, std::vector<std::string>& argv ) {
+            if( data == kASSET ) {
+                add::asset::issue_transaction( argv );
+            } else if( data == kDOMAIN ) {
+                add::domain::issue_transaction( argv );
+            } else if( data == kACCOUNT ) {
+                add::account::issue_transaction( argv );
+            } else if( data == kPEER ) {
+                add::peer::issue_transaction( argv );
+            } else {
+                undefined_data_error( data );
+            }
+        }
+        void Remove( std::string& data, std::vector<std::string>& argv ) {
```

Comment:
https://github.com/hyperledger/iroha/pull/209#discussion_r106595443
I should repair another branch.